modules/core/server/routes/core.server.routes.js

```diff
@@ -4,13 +4,21 @@
  * Module dependencies.
  */
 var path = require('path'),
+    redirect = require("express-redirect"),
     facebookNotificationService = require(path.resolve('./modules/core/server/services/facebook-notification.server.service')),
     core = require('../controllers/core.server.controller'),
     userProfile = require(path.resolve('./modules/users/server/controllers/users.profile.server.controller')),
     tribes = require(path.resolve('./modules/tribes/server/controllers/tribes.server.controller'));
 
+
 module.exports = function (app) {
 
+  redirect(app);
+
+  app.redirect('/invite', '/signup');
+  app.redirect('/tribes/lgbt', '/tribes/lgbtq');
+  app.redirect('/tribes/vegans-vegetarians', '/tribes/veg');
```

Comment:
We could  handle routes in route configs so that they'll stay in their respective modules.

So on top of [`modules/tribes/server/routes/tribes.server.routes.js`](https://github.com/Trustroots/trustroots/blob/ae4d7a16b5b1ec021c020841dddbc8325f936cc1/modules/tribes/server/routes/tribes.server.routes.js) we can do this:

```js
  app.route('/tribes/lgbt').get(function (req, res) {
    res.redirect(301, '/tribes/lgbtq');
  });
```

`/invite` redirect could go to `modules/users/server/routes/users.server.routes.js`

What do you think?

Reply:
Yeah makes sense. But I'm not yet sure how to do this. Where does the `redirect` function go?

Reply:
`res.redirect` is built in Express.js: https://expressjs.com/en/4x/api.html#res.redirect

Just adding these three lines to routes.js should be enough, no need to do anything at `express.js`

Reply:
Yeah ideally tribe ones would be coming from the DB. Next iteration!

Reply:
On the other hand, having code related to a feature outside the feature's bulk of code itself will make someone else slower because they need to now look for that code. ;-)

This one is fine now, let's 🚢